non-critical-infra/modules/mailserver/mailing-lists.nix

```diff
@@ -8,6 +8,26 @@
   # an encrypted password. Run `nix run .#encrypt-email login -- --help` and
   # follow the instructions.
   mailing-lists = {
+    # nixcon.org
+    "orgateam@nixcon.org" = {
+      forwardTo = [
+        ../../secrets/nixcon-list-email-address.umbriel
+      ];
+    };
+
+    "foundation@nixcon.org" = {
+      forwardTo = [
+        ../../secrets/nixcon-list-email-address.umbriel
+      ];
+    };
+
+    "volunteers@nixcon.org" = {
+      forwardTo = [
+        ../../secrets/nixcon-list-email-address.umbriel
+      ];
+    };
```

Comment:
These ones seem removable. If you do keep the foundation one, should probably forward to `foundation@nixos.org` though.

Reply:
We should probably only keep what we still reference in the current year.

Reply:
Dropped both.

Reply:
Confirmed with zimbatm that neither Pretix nor Pretalx use any of these addresses.